### PR TITLE
blehr: Fix state notifications

### PR DESCRIPTION
**Pull request by prasad-alatkar**
Use conn_handle in ble_gattc_notify_custom() instead o incorrect notify_state

### CODE REVIEW

File: apps/blehr/src/main.c [hunk 1]
```diff
@@ -162,11 +165,17 @@ blehr_gap_event(struct ble_gap_event *event, void *arg)
...
+        }
+        else {
+          conn_handle = event->connect.conn_handle;
         }
+
         break;
 
     case BLE_GAP_EVENT_DISCONNECT:
         MODLOG_DFLT(INFO, "disconnect; reason=%d\n", event->disconnect.reason);
+        conn_handle = 0; // reset conn_handle
```
Comment (andrzej-kaczmarek) [inline apps/blehr/src/main.c]: 0 is a valid connection handle, we have symbol defined iirc in GAP for invalid conn handle
also do not use C++ style comments

Comment (prasad-alatkar) [reply apps/blehr/src/main.c]: Could you point me to the symbol definition `iirc` ? I could not find it gap services.

**Comment by rymanluk** [comment]
@prasad-alatkar I meant commit message and not PR comment. The thing is the it is commit message which stays in the git tree, and PR comment disappears, that is why we care about commit message.

**Comment by prasad-alatkar** [comment]
Hi @rymanluk , got it :+1:

**Pull request merged**